lib/services/execution_iframe.dart

```diff
@@ -84,7 +82,7 @@ var resultFunction = _result;
 
   String _decorateJavaScript(String javaScript, {String modulesBaseUrl}) {
     final String postMessagePrint = '''
-const testKey = '$testKey';
+var testKey = '$testKey';
```

Comment:
@RedBrogdon This was added so that this block of code can be called multiple times:

```
Uncaught SyntaxError: Identifier 'testKey' has already been declaredError: SyntaxError: Identifier 'testKey' has already been declared
```